dipy/viz/regtools.py

```diff
@@ -4,6 +4,20 @@
 plt, _, _ = optional_package("matplotlib.pyplot")
 
 
+def _tile_plot(imgs, titles, **kwargs):
+    """
+    Helper function
+    """
+    # Create a new figure and plot the three images
+    fig, ax = plt.subplots(1, len(imgs))
+    for ii, a in enumerate(ax):
+        a.set_axis_off()
+        a.imshow(imgs[ii], **kwargs)
+        a.set_aspect('auto', 'box')
```

Comment:
People may not want to set auto-aspect ratio (e.g. when using "plot_slices" the coronal and sagital slices look weird), maybe add it as an optional parameter?